core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/dot-edit-page-routing.module.ts

```diff
@@ -54,11 +54,13 @@ const dotEditPage: Routes = [
             {
                 path: 'experiments',
                 loadChildren: async () =>
-                    (
-                        await import(
-                            '../dot-experiments/dot-experiments-shell/dot-experiments-shell-routes'
-                        )
-                    ).DotExperimentsShellRoutes
+                    //TODO: move all the core-web/apps/dotcms-ui/src/app/view/components/_common
+                    // folder with components reused in experiments to a Library to
+                    // avoid this circular dependency
```

Comment:
is this done?

Reply:
No, its a big task, and we will do in another issues (components, pipes and directives)